docs/guides/hedgehog-engine/rangers/materials/advanced-mats.md

```diff
@@ -0,0 +1,91 @@
+---
+description: Advanced Material Editing guide for Sonic Frontiers
+---
+# Advanced Material Editing
+
+This guide will assume you have at least some understanding of how PBR texturing works.
+
+!!! info
+    This guide uses the following tools:
+
+    - [HedgeEdit](https://drive.google.com/file/d/1VDHTpPe1MNVlAYblLsDd3Oa1KpsLqLGs/view){ target="_blank"}
```

Comment:
Please do not use Drive links. I'd recommend placing HedgeEdit in a tools page for HedgeDocs. If I recall correctly, it's a level editor for all the hedgehog engine games, so you could place it in a category that matches that description.

Reply:
Ive made a tools page, but where would it link to? hedgeedit has been removed from the github it originates from. So all we really have are backups